conf/modules.config

```diff
@@ -67,318 +64,297 @@ process {
-                ]]
-            }
-        }
-
-        //
-        // Gridss
-        //
-
-        if("gridss" in callers){
-            withName: "^.*BAM_VARIANT_CALLING_GRIDSS:GRIDSS_GRIDSS\$" {
-                cpus   = { check_max( 8     * task.attempt, 'cpus'    ) }
-                memory = { check_max( 32.GB * task.attempt, 'memory'  ) }
-                ext.prefix = { "${meta.id}.gridss" }
-                ext.args = { [
-                    "--steps preprocess,assemble,call",
-                    "--otherjvmheap ${task.memory.toGiga() > 10 ? 4 : 2}g",
-                    "--jvmheap ${task.memory.toGiga() > 10 ? task.memory.toGiga() - 8 : 2}g"
-                ].join(" ") }
-                publishDir = [
-                    enabled: output_callers,
-                    overwrite: true,
-                    path: { "${params.outdir}/${meta.id}/gridss" },
-                    mode: params.publish_dir_mode,
-                    saveAs: { filename -> filename ==~ /^.*.vcf.gz$/ ? filename : null }
-                ]
-            }
+    //
+    // Whamg
+    //
+
+    // if("whamg" in callers){
+    //     withName: "^.*BAM_VARIANT_CALLING_WHAMG:WHAMG\$" {
+    //         ext.prefix = { "${meta.id}.whamg" }
+    //         ext.args = {[
+    //             meta.region ? "-r ${meta.region}" : "" ,
+    //             "-z"
+    //         ].join(' ').trim()}
+    //         publishDir = [
+    //             enabled: output_callers,
+    //             overwrite: true,
+    //             path: { "${params.outdir}/${meta.id}/whamg" },
+    //             mode: params.publish_dir_mode,
+    //             saveAs: { filename -> filename != "versions.yml" ? filename : null }
+    //         ]
+    //     }
+    // }
+
+    //
+    // Manta
+    //
+
+    withName: "^.*BAM_VARIANT_CALLING_MANTA:GAWK\$" {
+        ext.suffix = "bed"
+        ext.prefix = "contigs"
+        ext.args2 = '\'BEGIN {FS="\t"}; \$1 ~ /^chr[1-9XY][1-9]?\$/ {print \$1 FS "0" FS \$2 }\''
+    }
 
-            withName: "^.*BAM_VARIANT_CALLING_GRIDSS:TABIX_TABIX\$" {
-                publishDir = [
-                    enabled: output_callers,
-                    overwrite: true,
-                    path: { "${params.outdir}/${meta.id}/gridss" },
-                    mode: params.publish_dir_mode,
-                    saveAs: { filename -> filename.endsWith(".tbi") ? filename : null }
-                ]
-            }
+    withName: "^.*MANTA_.*\$" {
+        container = "quay.io/cmgg/manta:1.6.0"
+    }
 
-            withName: "^.*BAM_VARIANT_CALLING_GRIDSS:VIOLA\$" {
-                time   = { check_max( 20.h  * task.attempt, 'time') }
-                ext.prefix = { "${meta.id}.viola" }
-                ext.args = { meta.read_length ? "--read_length ${meta.read_length}" : ""}
-            }
+    withName: "^.*BAM_VARIANT_CALLING_MANTA:MANTA_GERMLINE\$" {
+        label = "process_high"
+        cpus  = { 20 * task.attempt }
```

Comment:
isn't 20 cpu by default a little overkill?

Reply:
Nope sadly it isn't :/ It takes very long with lower cpu counts. But this can be re-evaluated once the pipeline is stable